utils_cv/multi_object_tracking/convert_seq_vid.py

```diff
@@ -0,0 +1,83 @@
+import cv2
+import os
+import glob
+import argparse
+
+
+def vid_to_seq(seq_dir, vid_path, fps):
+    """
+
+    :param seq_dir:
+    :param vid_path:
+    :param fps:
+    :return:
+    """
+    vdo = cv2.VideoCapture()
+    vdo.open(vid_path)
+    
+    if not os.path.exists(seq_dir):
+        os.mkdir(seq_dir)
+
+    idx_frame = 0
+    while vdo.grab():
+        _, image = vdo.retrieve()
+        cv2.imwrite(os.path.join(seq_dir, "frame{:06d}.jpg".format(idx_frame)), image)
+        idx_frame += 1
+
+
+def seq_to_vid(seq_dir, vid_path, fps):
+    """
+
+    :param seq_dir:
+    :param vid_path:
+    :param fps:
+    :return:
+    """
+    img_array = []
+    files = glob.glob(os.path.join(seq_dir, '*.gif'))
+    files.extend(glob.glob(os.path.join(seq_dir, '*.png')))
+    files.extend(glob.glob(os.path.join(seq_dir, '*.jpg')))
+    
+    print("files:", files)
+    files = sorted(files)#, key = lambda x:int(x.split(".")[0]))
+    print("files sorted:", files)
+    for file in files:
+        img = cv2.imread(file)
+        height, width, layers = img.shape
+        size = (width, height)
+        img_array.append(img)
+    
+    out = cv2.VideoWriter(vid_path, cv2.VideoWriter_fourcc(*'MP4V'), fps, size)
+    for image in img_array:
```

Comment:
Looks like img_array isn't necessary at all. Why not just call .write() in the loop above?